client/App.jsx

```diff
@@ -9,7 +9,7 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      'product_id': '22124'
+      'product_id': '22122'
```

Comment:
Did you mean to change the product id?

Reply:
@matthewwrobel I was trying to find a product_id that gives me the best reviews to work with. Will be careful with committing and pushing this change going forward!